app/views/admin/conferences/show.html.haml

```diff
@@ -1,6 +1,6 @@
 %h1
   %span.fa.fa-tachometer
-  Dashboard for #{@conference.title}
+  Dashboard for #{@conference.title} ( by #{@conference.organization.name} )
```

Comment:
I'd prefer to have no spaces inside the parenthesis:
`(by #{@conference.organization.name})`